.env.example

```
@@ -0,0 +1,13 @@
+# GitHub Stats API
+# Generate at: https://github.com/settings/tokens
+# Required scopes: read:user, read:org
+GITHUB_TOKEN=ghp_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+GITHUB_USERNAME=your-github-username
+
+# Contact Form (Nodemailer SMTP)
+# For Gmail, use an App Password: https://support.google.com/accounts/answer/185833
+EMAIL_HOST=smtp.gmail.com
+EMAIL_PORT=587
+EMAIL_USER=your-email@gmail.com
+EMAIL_PASS=your-app-password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_PASS key should go before the EMAIL_PORT key

(UnorderedKey)